plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/VariationSelectorAttributeOptions.php

```diff
@@ -161,15 +177,15 @@ function () {
 				'<input type="radio" %s/>',
 				$this->get_normalized_attributes(
 					array(
-						'class'                  => 'wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill-input',
-						'name'                   => $attribute_slug,
-						'value'                  => $attribute_term['value'],
-						'data-wp-bind--checked'  => 'state.isOptionSelected',
-						'data-wp-bind--disabled' => 'state.isOptionDisabled',
-						'data-wp-watch'          => 'callbacks.watchSelected',
-						'data-wp-on--click'      => 'actions.handlePillClick',
-						'data-wp-on--keydown'    => 'actions.handleKeyDown',
-						'data-wp-context'        => array(
+						'class'                     => 'wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill-input',
+						'name'                      => $attribute_slug,
+						'value'                     => $attribute_term['value'],
+						'data-wp-bind--checked'     => 'state.isOptionSelected',
+						$wp_interactivity_directive => 'state.isOptionDisabled',
```

Comment:
Would this be equivalent to?

```suggestion
						'data-wp-bind--hidden' => 'hide' === $disabled_attributes_action ? 'state.isOptionDisabled' : null,
						'data-wp-bind--disabled' => 'state.isOptionDisabled',
```

This way, we could get rid of the `switch` above. I believe it will make the code a bit cleaner.

Reply:
This was because before, it was possibly grayed-out but still clickable, now that we removed this option, I will simplify it.